src/main/java/ru/skypro/homework/dto/CreateComment.java

```diff
@@ -4,10 +4,13 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+/**
+ * DTO создания комментария.
+ */
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
-public class PictureDto {
-    private String idPicture;
-    private byte[] data;
+public class CreateComment {
```

Comment:
```suggestion
public class CreateOrUpdateComment {
```
см. спецификацию внимательнее, эта dto используется не только для создания, но и обновления (см. комментарий)